### DISCUSSION TITLE
convert TIF directory to JPEG direcory

### DISCUSSION THREAD

**Discussion by SylvainArd** [ImageMagick/ImageMagick#5093] [Help]
Hello !
I have a directory A with about 50 TIF images, I want to convert them all into JPEG files in the B directory please.
I precise that I am under Windows 10.
Thank you
Best regards

**Comment by SylvainArd** [comment]
I precise that I want a BATCH command

**Comment by snibgo** [comment]
I suggest "magick mogrify", like this:
```
magick mogrify -path directoryB -format jpg directoryA\*.tif
```
See https://imagemagick.org/script/mogrify.php

**Comment by SylvainArd** [comment]
it works thank you very much !
Sylvain Ard
0549507724
0778380991
***@***.***
http://sylvain-ard.fr
Entreprise individuelle SIRET : 80079243400022
Appt 26 Bât A Résidence Le Patio
83 rue de la Bugellerie
86000 Poitiers
